extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/querykit/WindowOperatorQueryKit.java

```diff
@@ -233,7 +210,7 @@ public QueryDefinition makeQueryDefinition(
                            .shuffleSpec(nextShuffleSpec)
                            .processorFactory(new WindowOperatorQueryFrameProcessorFactory(
```

Comment:
I feel like the full stage should be build by the `WindowStage`

Reply:
With the latest PR changes where I added the `WindowStages` class, this is being done there now. Appreciate your inputs if this makes sense to you. Thanks!